src/components/PupilViews/PupilIntro/PupilIntro.view.tsx

```diff
@@ -92,16 +111,34 @@ export const PupilViewsIntro = (props: PupilViewsIntroProps) => {
     />
   );
 
+  const handleBottomButtonClick = () => {
+    if (sectionResults.intro?.isComplete) {
+      const nextSection =
+        lessonReviewSections.find(
+          (section) => !sectionResults[section]?.isComplete,
+        ) ?? "review";
+      trackSectionStarted(nextSection);
```

Comment:
soon we will want component coordinates with this information. Ie. that the user got to the next section using this particular button

Reply:
love comments like this. soon?! how soon? in this pr soon?